elfi/methods/post_processing.py

```diff
@@ -0,0 +1,233 @@
+"""
+Post-processing for posterior approximations from other ABC algorithms.
+
+See the review paper:
+
+Fundamentals and Recent Developments in Approximate Bayesian Computation
+Lintusaari et. al
+Syst Biol (2017) 66 (1): e66-e82.
+https://doi.org/10.1093/sysbio/syw077
+
+for more information.
+"""
+
+from sklearn.linear_model import LinearRegression
+import numpy as np
+
+from . import results
+
+
+__all__ = ('LinearAdjustment', 'adjust_posterior')
```

Comment:
The namespace directly under `elfi` should in essence only contain methods that users use directly. You can put the `LinearAdjustment` under e.g. `elfi.post_processing` module. Would be nice if you could specify strings as optional way to specify the method, for this e.g. just 'linear', since it's a very well known and common method.